include/xsimd/arch/xsimd_neon64.hpp

```diff
@@ -592,6 +592,18 @@ namespace xsimd
             return vabsq_f64(rhs);
         }
 
+        /**************
+         * reciprocal *
+         **************/
+
+        template <class A>
+        inline batch<double, A>
+        reciprocal(const batch<double, A>& x,
+                   kernel::requires_arch<neon64>) noexcept
+        {
+            return vrecpeq_f64(x);
```

Comment:
ok, so SSE only has float specialization, while neon has the double one too, this should be reflected in the `xsimd_api.h` file

Reply:
There's an AVX512F-provided intrinsic, but it requires AVX512VL as well: https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=mm_rcp1&ig_expand=5571

I've patched it by requiring SFINAE if T is not a floating point type, is that OK with you?